src/components/reportPage/UploadBox.jsx

```diff
@@ -1,31 +1,48 @@
-import React, { useCallback, useEffect } from "react";
+import React, { useCallback, useEffect, useState } from "react";
 import { useDropzone } from "react-dropzone";
 import "./upload.css";
 import CloudIcon from "./cloudIcon";
 const UploadBox = (props) => {
   const onDrop = useCallback((files) => {
     props.setFiles((existingFiles) => [...existingFiles, ...files]);
   }, []);
-  const { setAlert, fileLimit } = props;
+
+  const { setAlert, fileLimit, currFiles } = props;
   const MAX_FILE_ALERT = `Please upload only up to ${fileLimit} valid files.`;
   const INVALID_FILETYPE = "You are trying to upload invalid file types.";
+
   const onDropRejected = (fileRejections) => {
     const rejectFiles = fileRejections.length;
     if (rejectFiles) {
+      console.log(rejectFiles);
```

Comment:
Is this `console.log` necessary?

Reply:
No, it's not, I was using it for testing purposes. Will remove it.